lib/api/graphql/graphql_client_provider.dart

```diff
@@ -32,7 +32,7 @@ class GraphQLClientProvider {
         _allLink = _httpLink;
       }
       _client = GraphQLClient(
-        cache: GraphQLCache(),
+        cache: GraphQLCache(store: HiveStore()),
```

Comment:
I can't imagine it's so easy to support offline-mode for graph-ql 😂

Reply:
it's not specifically graphql but it's about the client lib you are using 🙏  I believe there must be something similar for HTTP too; unlike our native android (maybe they have too)